CHANGELOG.md

```diff
@@ -6,26 +6,30 @@ This file is used to list changes made in each version of the aws-parallelcluste
 
 2.10.0
 -----
+**ENHANCEMENTS**
+- Add new `all_or_nothing_batch` configuration parameter for `slurm_resume`. If set to `True`, RunInstances calls will
+  not succeed unless all capacity for the batch can be launched.
 
 **CHANGES**
-- Optimized the retrieval of nodes info from Slurm scheduler.
-- Increase default timeout for Slurm commands submitted by clustermgtd and computemgtd from 10 to 30 seconds.
+- CentOS 6 is no longer supported.
+- Optimize retrieval of nodes info from Slurm scheduler.
+- Improve retrieval of instance type info by using `DescribeInstanceType` API.
+- Increase default timeout for Slurm commands submitted by `clustermgtd` and `computemgtd` from 10 to 30 seconds.
```

Comment:
Increase timeout from 10 to 30 seconds when `clustermgtd` and `computemgtd` daemons invoke Slurm commands